standards/core/relay-sharding.md

```diff
@@ -34,19 +34,28 @@ This document also covers discovery of topic shards.
 
 ## Named Sharding
 
+> **_Note:_** As of 2025-07-24 named sharding has been deprecated from all Waku implementations.
+It is still described here as background for static and automatic sharding.
+
 _Named sharding_ offers apps to freely choose pubsub topic names.
 It is RECOMMENDED for App protocols to follow the naming structure detailed in [23/WAKU2-TOPICS](https://github.com/vacp2p/rfc-index/blob/main/waku/informational/23/topics.md).
 With named sharding, managing discovery falls into the responsibility of apps.
+For this reason it is NOT RECOMMENDED that app protocols and Waku implementations make use of named sharding.
 
 From an app protocol point of view, a subscription to a content topic `waku2/xxx` on a shard named /mesh/v1.1.1/xxx would look like:
 
 `subscribe("/waku2/xxx", "/mesh/v1.1.1/xxx")`
 
 ## Static Sharding
 
-_Static sharding_ offers a set of shards with fixed names.
+_Static sharding_ is an extension of named sharding that offers a set of shards with _fixed_ names.
 Assigning content topics to specific shards is up to app protocols,
 but the discovery of these shards is managed by Waku.
+This is the RECOMMENDED default format for shards chosen by an app protocol.
+It is RECOMMENDED that, for simplification of configuration and various APIs,
+all app-level protocols only interact with `cluster` and `shard`
+and never the fully-formed pubsub topic,
+which is a concern internal to the Waku core.
```

Comment:
maybe `libraries` instead of `core`, I believe word `core` might be misunderstood by people who didn't participate in our discussions around APIs etc.

```suggestion
which is a concern internal to the Waku libraries.
```

Reply:
`implementations` might be most accurate.